### PR TITLE
hexane refractor and rework

### PR DESCRIPTION
**Pull request by Ghilker**
## About The Pull Request
Hexane no longer grant the ability to hear dchat (yep it was a bad idea)
Hexane now gives the resist heat trait to fight fires better but can give you brain damage (small amount behind a 25% chance) and a lot of hallucinations

## Why It's Good For The Game
hearing dchat bad + refractor good

## Changelog
:cl:
refractor: hexane properties are on the reagent and not the breathing
del: hexane no longer grand you dchat hearing
add: hexane makes you heat resistant (can still die due to pressure)
del: Hexane no longer deals brain damage 
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -1401,14 +1401,20 @@
...
 /datum/reagent/hexane/on_mob_end_metabolize(mob/living/L)
 	L.remove_movespeed_modifier(/datum/movespeed_modifier/reagent/halon)
 	REMOVE_TRAIT(L, CHANGELING_HIVEMIND_MUTE, type)
-	REMOVE_TRAIT(L, TRAIT_SIXTHSENSE, type)
+	REMOVE_TRAIT(L, TRAIT_RESISTHEAT, type)
 	return ..()
 
+/datum/reagent/hexane/on_mob_life(mob/living/L)
+	. = ..()
+	L.hallucination += 150
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm:1414]: um

you do know that this is a duration, not an intensity, right? each tick (2 seconds, I think) of this reagent beign in your system is going to apply 150 ticks' worth (5 minutes' worth if delta \_time hasn't changed things up) of hallucinations.

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 2]
```diff
@@ -1401,14 +1401,20 @@
 	. = ..()
 	L.add_movespeed_modifier(/datum/movespeed_modifier/reagent/halon)
 	ADD_TRAIT(L, CHANGELING_HIVEMIND_MUTE, type)
-	ADD_TRAIT(L, TRAIT_SIXTHSENSE, type)
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm:1404]: how the hell did a gas that let you hear deadchat get approved in the first place?

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 1 above)
Comment (Ghilker) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm:1414]: ohhhhhh i thought it was some kind of intensity amount... what should be the right thing to do? just something like + 10 ?

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 2 above)
Comment (LemonInTheDark) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm:1404]: Not my finest moment

**Comment by LemonInTheDark** [comment]
Are you sure about the brain damage? It'll discourage the use of the gas for fighting fires, etc

**Comment by Ghilker** [comment]
I'm unsure to be honest, i can remove it if it's too much

**Comment by LemonInTheDark** [comment]
Well what should the gas be used for?

**Comment by Ghilker** [comment]
yeah i'll remove the brain damage, it's dumb

**Comment by ExcessiveUseOfCobblestone** [comment]
Brain damage would be fine if it wasn't 3 on rng lol

**Comment by LemonInTheDark** [comment]
Talk to cobby about that, fix the changelog, and we'll be good

**Pull request merged**